tinytroupe/agent/grounding.py

```
@@ -0,0 +1,272 @@
+        """
+        Adds a path to a folder with files used for grounding.
+        """
+
+        if folders_paths is not None:
+            for folder_path in folders_paths:
+                try:
+                    logger.debug(f"Adding the following folder to grounding index: {folder_path}")
+                    self.add_folder(folder_path)
+                except (FileNotFoundError, ValueError) as e:
+                    print(f"Error: {e}")
+                    print(f"Current working directory: {os.getcwd()}")
```

Ruff (0.8.2)

Undefined name `os`

(F821)